test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs

```diff
@@ -139,7 +139,7 @@ public void CannotStartConnectionWithInvalidTransportType(TransportType requeste
 
             var transport = new Mock<ITransport>();
             transport.Setup(t => t.StopAsync()).Returns(async () => { await releaseDisposeTcs.Task; });
-            var connection = new HttpConnection(new Uri("http://fakeuri.org/"), new TestTransportFactory(transport.Object), loggerFactory: null, httpMessageHandler: mockHttpHandler.Object);
+            var connection = new HttpConnection(new Uri("http://fakeuri.org/"), transportFactory: new TestTransportFactory(transport.Object), loggerFactory: null, httpMessageHandler: mockHttpHandler.Object);
```

Comment:
I don't think you need the name - the type we new up tells you what you need to know. (I don't care too much though)